### PR TITLE
Adding a few missing words

### LINKED ISSUE #245: missing important word in explanation for Variable Declarations > Block-scoped variable capturing
"When we first touched on the idea of variable capturing with var declaration, we briefly went into how variables act once captured. To give a better intuition of this, each time a scope is run, it creates an “environment” of variables. That environment and its **[i dunno what goes here]** can exist even after everything within its scope has finished executing."

I'd do a pull request but I don't know what was intended be plugged in there.

### PR DESCRIPTION
**Pull request by drmacm**
Fixes #245 (and another 2 small unreported issues)

### CODE REVIEW

File: pages/Interfaces.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ mySearch = function(source: string, subString: string) {
 }
 ```
 
-For function types to correctly type-check, the name of the parameters do not need to match.
+For function types to correctly type-check, the name of the parameters does not need to match.
```
Comment (DanielRosenwasser) [inline pages/Interfaces.md]: The correct fix is actually to make it "names of the parameters"

**Comment by DanielRosenwasser** [comment]
Thanks @drmacm, I just left one comment to address. Otherwise looks good.

**Comment by drmacm** [comment]
Sure, I will fix it properly and resubmit immediately.

**Comment by DanielRosenwasser** [comment]
Awesome, thanks for the PR!

**Pull request merged**